Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,364 @@
+| Proxy cache                         | ✓                                             | ✓ Feature is in the technology preview stage (non production ready) | ✗                            |
+| Quota management                    | ✓ Based on storage consumption or image count | ✓ Based on storage consumption                                      | ✗                            |
+| Garbage collection                  | ✓                                             | ✓                                                                   | ✗                            |
+| Retention policy                    | ✓ Multiple tag retention rules                | ✓ Only tag expiration rules                                         | ✗                            |
+| Additional supported artifacts      | ✓ non-OCI Helm charts (ChartMuseum)           | ✗ (only OCI artifacts)                                              | ✓ Maven, YUM                 |
+| Integration possibilities           | ✓ Dragonfly (P2P), Kraken (P2P)               | ✗                                                                   | ✓ Harbor, Nydus, eStargz     |
+
+## Conclusion
+
+A wide range of open-source container registry projects (Quay, Harbor, Dragonfly,
+Keppel, Portus, Kraken, etc.) has been carefully evaluated based on the two main
+factors: the open-source health and range of supported features.
+
+The open-source software health is crucial and container registry implementation should
+pass it to be SCS-compliant. The OSS health check evaluates several important metrics
+of an open source software like whether the code/community/development/design is
+fully open or whether the project's maturity, security, and activity are on the desired
+level. This check also evaluates the lock-in risk due to possible single points of
+failure or internal project conflicts and several other aspects.
+Overall, three projects passed the OSS health checks:
+- [Harbor][harbor]
+- [Project Quay][projectquay]
+- [Dragonfly][dragonfly]
+
+The above projects were then evaluated from the "supported features" perspective.
+The [Required and desirable features check](#required-and-desirable-features-check)
+did an overview of the feature set of open-source container registry implementations
+and map SCS requirements (and nice-to-haves) against it. The list of required features
+is quite long and contains features that are primarily focused on security
+(authentication, vulnerability scanning, content trust, and validation, etc.),
+scalability (HA mode, registry replication, p2p integration, etc.) and visibility
+(monitoring), see the full list [here](#required-and-desirable-features-check).
+These requirements should ensure that the selected container registry implementation
+could be offered by the CSPs as a secure and enterprise-ready solution.
+
+Let's discuss and compare the projects Dragonfly, Quay, and Harbor in the following
```

Comment:
Starting the sentence with `Let's` is a stark change in style and feels distinctly out-of-place within the wider context of this document.